### PR TITLE
Add acceptance testing for 4.2 and 4.4

### PR DESCRIPTION
**Pull request by baurmatt**
Test current version of Zabbix.

### CODE REVIEW

**Comment by baurmatt** [comment]
Travis for Ubuntu 16.04:

```
Error: Unable to connect to PostgreSQL server! (:5432)

  Error: /Stage[main]/Postgresql::Server::Service/Postgresql_conn_validator[validate_service_is_running]/ensure: change from 'absent' to 'present' failed: Unable to connect to PostgreSQL server! (:5432)
```

I *think* the error is unrelated because:
1. Works on my machine ;)
2. Happens on master as well
3. Works for all other OSs

**Comment by alexjfisher** [comment]
@baurmatt Build on master has been fixed.  Wanna rebase?

File: spec/acceptance/agent_spec.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 require 'spec_helper_acceptance'
 
-['2.4', '3.2', '3.4', '4.0'].each do |version|
+['2.4', '3.2', '3.4', '4.0', '4.2', '4.4'].each do |version|
```
Comment (bastelfreak) [inline spec/acceptance/agent_spec.rb:3]: this only adds them for the agent. Do you also want to do this for the server? (maybe also have a look at https://github.com/voxpupuli/puppet-zabbix/pull/674 )

Comment (baurmatt) [reply spec/acceptance/agent_spec.rb:3]: I couldn't find a similar version specific testing for the server resources and I'm currently short on time for restructuring the server_spec.rb File. Sorry! :)

**Pull request merged**